README.md

```diff
@@ -3,13 +3,13 @@
 ## Definitions
 Define the following: 
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
-- Activation Record/Call
-- Infinite Recursion/Stack Overflow/Stack too deep
-- Tail Recursion
+- Recursion: a function that calls itself (see recursion)
+- Recursive Case: The function that is called within itself, should bring you closer to the base case.
+- Base Case: the base case is what stops the recursion from infinite recursion
+- Activation Chain/Stack - This still isn't clear to me
+- Activation Record/Call - This still isn't clear to me
```

Comment:
An activiation record is a single call of the recursive method. So one "bubble" if you will.